packages/fe/test/services/utils.ts

```diff
@@ -52,7 +53,7 @@ export const getViewCompletionProvider = (
       content,
       offset
     );
-    const context = await getContext(documentPath);
+    const context = (await getContext(documentPath)) as Context;
```

Comment:
I don't think its safe to assume that `context` is not going to of type `Error`.  If its an error then we'll have a runtime error when trying to work with a different type of object.

Reply:
yes, but in case of test, it is fine to make that assumption.